client-v2/src/main/java/com/clickhouse/client/api/enums/ProxyType.java

```diff
@@ -0,0 +1,5 @@
+package com.clickhouse.client.api.enums;
+
+public enum ProxyType {
+    IGNORE, DIRECT, HTTP, SOCKS;
```

Comment:
I suggest to reuse `java.net.Proxy.Type`. 

Current class I think has some extra constants
- IGNORE - ignoring a proxy is the same as DIRECT connection 
- DIRECT - creates confusion in our case - because our API requires to add proxy and adding "direct" proxy has no effect.